.github/workflows/ci.yml

```diff
@@ -101,5 +101,8 @@ jobs:
       - name: Run bulloak check
         run: yarn lint:bulloak
 
+      - name: Ensure auto-generated smocks are up to date
+        run: yarn smock && git diff --exit-code -- test/smock
```

Comment:
hmmmm ok, but doesn't it check for diff in non-existent contracts?

Reply:
that's correct, it doesn't check on untracked files. If a file were to be tracked by git and then _removedd_ by `yarn smock`, this would cause a CI error, but that's not something likely

it won't report an error if `smock` generates a file which git does not track